CHANGELOG.md

```diff
@@ -204,10 +204,13 @@ v0.37.0-rc.0 (2023-10-05)
 
 - `prometheus.operator.probes` no longer ignores relabeling `rule` blocks. (@sberz)
 
+- Migrate `Check Linux/Windows build image` to GitHub Actions. (@hainenber)
```

Comment:
While we appreciate your work here, the CHANGELOG is only for features that impact end users of the agent, so I'll be removing this line, sorry 😞

Reply:
Yeah, no worries :D